src/features/rewards/infoCard/style.ts

```diff
@@ -41,3 +41,12 @@ export const StyledFigure = styled<{}, 'figure'>('figure')`
   margin: 10px auto 20px;
   height: 80px;
 `
+export const StyledGrid = styled<{}, 'div'>('div')`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+export const StyledColumn = styled<{}, 'div'>('div')`
+  display: inline-block;
+  padding: 0 15px;
```

Comment:
padding should be reduced to 10px so that we are not effecting layout in full width, but still preserving spacing when resizing